cmsplugin_filer_image/templates/cmsplugin_filer_image/image.html

```diff
@@ -1,46 +1,50 @@
-{% load thumbnail filer_tags filer_image_tags sekizai_tags %}{% spaceless %}
-{% addtoblock "css" %}<link rel="stylesheet" type="text/css" href="{{ STATIC_URL }}cmsplugin_filer_image/css/filer_image.css" media="screen, projection" />{% endaddtoblock "css" %}
+{% load thumbnail filer_tags filer_image_tags sekizai_tags %}
+{% spaceless %}
+{% addtoblock "css" %}
+<link rel="stylesheet" type="text/css" href="{{ STATIC_URL }}cmsplugin_filer_image/css/filer_image.css" media="screen, projection"/>
+{% endaddtoblock %}
 
-{% with d1='0123456789'|make_list|random d2='0123456789'|make_list|random d3='0123456789'|make_list|random d4='0123456789'|make_list|random %}
-  {% with rnd='imagePlugin_'|add:d1|add:d2|add:d3|add:d4 %}
+{% addtoblock "js" %}
+{% if event_tracking %}
+<script type="text/javascript">
+ (function($) {
```

Comment:
this can be merged with the <script> tag bellow.
Also adding an id attribute to the script tag will help with debugging as on a typical Bento page you will have multiple consecutive such scripts coming from different plugins/snippets
ex:
id="script_img_plg_{{rnd}}"